packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -1,30 +1,21 @@
 <template>
 	<div :class="formGroupStyle">
-		<div
-			:class="decreaseButtonStyle"
-			@click="setCurrentValue(currentValue - step)"
-		>
+		<div :class="decreaseButtonStyle" @click="decrementValue">
 			<i class="yoco">minus</i>
 		</div>
 		<input
 			ref="inputNumber"
```

Comment:
i think u can remove this ref (not used)

Reply:
kk. although we might keep it - its useful for testing because u can select the component by ref name.